workflow-templates/im-build-db-ci.yml

```diff
@@ -0,0 +1,186 @@
+name: Build Database CI
+on: 
+  push:
+    branches: [ master ]
+  pull_request:
+    branches: [ master ]
+env:
+  DB_NAME: '' # TODO: Fill in your repo's database name
+  DB_SERVER_NAME: 'localhost,1433'
+  DEFAULT_BRANCH: 'main'
```

Comment:
We've got some mismatch here with the branch names.  The plan is to move everyone to main, not master, so I'd prefer to update line 4 and 6 with main and have it match line 10.